### ISSUE TITLE
Improve map location relevance

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#193]
## User story
1. As a User
2. I want the map / search to be more specific to germany
3. So that I can focus on the relevant parts

## Acceptance criteria
* #194 
* #196 
* #201 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
